mods/d2k/rules/vehicles.yaml

```diff
@@ -344,22 +229,250 @@ MISSILETANK:
+	WithMuzzleFlash:
+	AutoTarget:
+	Explodes:
+		Weapon: UnitExplodeScale
+		EmptyWeapon: UnitExplodeScale
+	Selectable:
+		Bounds: 44,38,0,0
+	LeavesHusk:
+		HuskActor: devast.husk
+	AttractsWorms:
+		Intensity: 700
+
+raider:
+	Inherits: ^Vehicle
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 10
+		Prerequisites: ~light.ordos
+	Valued:
+		Cost: 300
+	Tooltip:
+		Name: Raider Trike
+		Description: Improved Scout\n Strong vs Infantry, Light Vehicles
+	Selectable:
+		Bounds: 24,24
+	Health:
+		HP: 110
+	Armor:
+		Type: Light
+	Mobile:
+		ROT: 10
+		Speed: 149
+	RevealsShroud:
+		Range: 7c0
+	WithMuzzleFlash:
+	Armament:
+		Weapon: HMGo
+		LocalOffset: 256,0,128
+		MuzzleSequence: muzzle
+	AttackFrontal:
+	AutoTarget:
+	Explodes:
+		Weapon: UnitExplodeTiny
+		EmptyWeapon: UnitExplodeTiny
+	AttractsWorms:
+		Intensity: 420
+
+stealthraider:
+	Inherits: raider
+	Buildable:
+		Prerequisites: ~light.ordos, hightech, ~techlevel.medium
+		BuildPaletteOrder: 30
+	Valued:
+		Cost: 400
+	Tooltip:
+		Name: Stealth Raider Trike
+		Description: Invisible Raider Trike\n Strong vs Infantry, Light Vehicles
+	Cloak:
+		InitialDelay: 45
+		CloakDelay: 90
+		CloakSound: STEALTH1.WAV
+		UncloakSound: STEALTH2.WAV
+	AutoTarget:
+		InitialStance: HoldFire
+	-MustBeDestroyed:
+
+deviatortank:
+	Inherits: ^Tank
+	Valued:
+		Cost: 1000
+	Tooltip:
+		Name: Deviator
+		Description: Fires a warhead which changes\nthe allegiance of enemy vehicles
+	Buildable:
+		Queue: Armor
+		BuildPaletteOrder: 50
+		Prerequisites: ~heavy.ordos, research, ~techlevel.high
+	Mobile:
+		ROT: 3
+		Speed: 64
+	Health:
+		HP: 125
+	Armor:
+		Type: Light
+	RevealsShroud:
+		Range: 5c0
 	RenderUnit:
-		Image: missiletank.destroyed
-	TransformOnCapture:
-		IntoActor: missiletank
+	Armament:
+		Weapon: NerveGasMissile
+		LocalOffset: -299,0,85
+	AttackFrontal:
+	AutoTarget:
+		InitialStance: Defend
+	Explodes:
+		Weapon: UnitExplodeSmall
+		EmptyWeapon: UnitExplodeSmall
+	Selectable:
+		Bounds: 30,30
+	LeavesHusk:
+		HuskActor: deviatortank.husk
+	AttractsWorms:
+		Intensity: 600
 
-MISSILETANK.starport:
-	Inherits: MISSILETANK
+^combat:
+	Inherits: ^Tank
 	Buildable:
-		Prerequisites: starport
-		Queue: Starport
+		Queue: Armor
+		BuildPaletteOrder: 40
 	Valued:
-		Cost: 1250
+		Cost: 700
+	Tooltip:
+		Name: Combat Tank
+		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry, Aircraft\n \n  Atreides:      +Range\n  Harkonnen: +Health\n  Ordos:         +Speed
+	Health:
+		HP: 350
+	Armor:
+		Type: Heavy
+	Mobile:
+		Speed: 64
+		ROT: 6
+		Crushes: crate, infantry
+	RevealsShroud:
+		Range: 8c0
+	Turreted:
+		ROT: 6
+		RealignDelay: 0
+	Armament:
+		Weapon: 90mm
+		Recoil: 128
+		RecoilRecovery: 32
+		LocalOffset: 256,0,0
+		MuzzleSequence: muzzle
+	AttackTurreted:
+	WithMuzzleFlash:
+	RenderUnit:
+	WithTurret:
+	AutoTarget:
+	Explodes:
+		Weapon: UnitExplodeSmall
+		EmptyWeapon: UnitExplodeSmall
+	Selectable:
+		Bounds: 30,30
+	AttractsWorms:
+		Intensity: 520
+
+combata:
```

Comment:
why not call these combattank.atreides, combattank.ordos etc. to match the other actor names?